Source/WebCore/platform/graphics/avfoundation/objc/ImageDecoderAVFObjC.mm

```diff
@@ -432,15 +438,22 @@ SampleFlags flags() const override
         return false;
     }
 
-    if (m_imageRotationSession)
+    if (m_imageRotationSession) {
         pixelBuffer = m_imageRotationSession->rotate(pixelBuffer.get());
+        setOwnershipIdentityForCVPixelBuffer(pixelBuffer.get(), m_resourceOwner);
+    }
 
     CGImageRef rawImage = nullptr;
     if (noErr != VTCreateCGImageFromCVPixelBuffer(pixelBuffer.get(), nullptr, &rawImage)) {
         RELEASE_LOG_ERROR(Images, "ImageDecoderAVFObjC::storeSampleBuffer(%p) - could not create CGImage from pixelBuffer", this);
         return false;
     }
 
+    // FIXME: The pixel buffer being passed to the VTCreateCGImageFromCVPixelBuffer may
```

Comment:
Nit: A FIXME should have an associated bugzilla ID in it so we don't forget to fix it.